CHANGELOG.md

```diff
@@ -1,23 +1,235 @@
+- `-Zcargo-lints`: Keep lints updated and sorted.
+  [#14030](https://github.com/rust-lang/cargo/pull/14030)
+- cargo-update: Track the behavior of `--precise <prerelease>`.
+  [#14013](https://github.com/rust-lang/cargo/pull/14013)
+
+### Documentation
+
+- contrib: Suggest atomic commits with separate test commits.
+  [#14014](https://github.com/rust-lang/cargo/pull/14014)
+
+### Internal
+
+- Remove the temporary `__CARGO_GITOXIDE_DISABLE_LIST_FILES` environment variable.
+  [#14036](https://github.com/rust-lang/cargo/pull/14036)
+- Update dependencies.
+  [#13995](https://github.com/rust-lang/cargo/pull/13995)
+  [#13998](https://github.com/rust-lang/cargo/pull/13998)
+  [#14037](https://github.com/rust-lang/cargo/pull/14037)
+
 ## Cargo 1.80 (2024-07-25)
-[b60a1555...HEAD](https://github.com/rust-lang/cargo/compare/b60a1555...HEAD)
+[b60a1555...rust-1.80.0](https://github.com/rust-lang/cargo/compare/b60a1555...rust-1.80.0)
 
 ### Added
 
+- 🎉 Stabilize `-Zcheck-cfg`! This by default enables rustc's checking of
+  conditional compilation at compile time, which verifies that the crate is
+  correctly handling conditional compilation for different target platforms or
+  features. Internally, cargo will be passing a new command line option
+  `--check-cfg` to all rustc and rustdoc invocations.
+
+  A new build script invocation
+  [`cargo::rustc-check-cfg=CHECK_CFG`](https://doc.rust-lang.org/nightly/cargo/reference/build-scripts.html#rustc-check-cfg)
+  is added along with this stabilization, as a way to add custom cfgs to the
+  list of expected cfg names and values.
+
+  If a build script is not an option for your package, Cargo provides a config
+  [`[lints.rust.unexpected_cfgs.check-cfg]`](https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html#check-cfg-in-lintsrust-table)
+  to add known custom cfgs statically.
+
+  ([RFC 3013](https://github.com/rust-lang/rfcs/blob/master/text/3013-conditional-compilation-checking.md))
+  ([docs](https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html))
+  [#13571](https://github.com/rust-lang/cargo/pull/13571)
+  [#13865](https://github.com/rust-lang/cargo/pull/13865)
+  [#13869](https://github.com/rust-lang/cargo/pull/13869)
+  [#13884](https://github.com/rust-lang/cargo/pull/13884)
+  [#13913](https://github.com/rust-lang/cargo/pull/13913)
+  [#13937](https://github.com/rust-lang/cargo/pull/13937)
+  [#13958](https://github.com/rust-lang/cargo/pull/13958)
+
+- 🎉 cargo-update: Allows `--precise` to specify a yanked version of a package,
+  and will update the lockfile accordingly.
+  [#13974](https://github.com/rust-lang/cargo/pull/13974)
+
 ### Changed
 
+- ❗️ manifest: Disallow `[badges]` to inherit from `[workspace.package.badges]`.
+  This was considered a bug and should inherit from `[workspace.badges]`.
+  Keep in mind that `[badges]` is effectively deprecated.
+  [#13788](https://github.com/rust-lang/cargo/pull/13788)
```

Comment:
> Even if someone were to try to guess to use this, it is inconsistent with how inheritance works because this should inherit from workspace.badges instead of workspace.package.badges

I think that line from the PR was misinterpreted.  It meant to say "the pattern for how to inherit would have us use `workspace.badges`.  This does not mean that `workspace.badges` is supported